Source/WebKit/WebProcess/cocoa/LaunchServicesDatabaseManager.mm

```diff
@@ -81,7 +81,12 @@
 void LaunchServicesDatabaseManager::waitForDatabaseUpdate()
 {
     auto startTime = MonotonicTime::now();
-    bool databaseUpdated = waitForDatabaseUpdate(5_s);
+#ifdef NDEBUG
+    auto waitTime = 5_s;
```

Comment:
Could these be `constexpr`?

Reply:
> Could these be `constexpr`?

Ah, good point! I will follow up with a patch for this.